app/views/projects/show.html.erb

```diff
@@ -89,19 +93,15 @@
   </table>
   <%= render partial: "projects/import_export" %>
 
-  <div class="btn-group">
-    <%= link_to "Edit or Delete Project", edit_project_path(@project.id), class: "button magenta"  %>
-
-    <% unless @project.parent_id %>
-      <%= link_to 'Add Sub-Project', project_new_sub_project_path(@project.id), id:"subproject", class: "button magenta" %>
-    <% end %>
-    <%= link_to 'Clone Project', duplicate_project_path(@project.id), method: :post, disable_with: 'Cloning...', class: "button magenta" %>
+  <div class="btn-group actions">
+    <%= link_unless_archived(@project, "Edit or Delete Project", edit_project_path(@project), color: "magenta") %>
+    <%= link_unless_archived(@project, "Add Sub-Project", project_new_sub_project_path(@project), color: "magenta") unless @project.parent_id %>
+    <%= link_unless_archived(@project, "Clone Project", duplicate_project_path(@project), method: :post, color: "magenta") %>
```

Comment:
I also think that cloning should probably be allowed even if archived, it's not modifying the project in any way and might be useful to clone an old project